zookeeper-server/src/main/java/org/apache/zookeeper/cli/ReconfigCommand.java

```diff
@@ -152,7 +153,7 @@ public boolean exec() throws CliException {
             }
 
             byte[] curConfig = ((ZooKeeperAdmin) zk).reconfigure(joining, leaving, members, version, stat);
-            out.println("Committed new configuration:\n" + new String(curConfig));
+            out.println("Committed new configuration:\n" + new String(curConfig, StandardCharsets.UTF_8));
```

Comment:
This should probably just be two `println` statements instead of appending a newline and using string concatenation.